bindings/java/src/main/com/apple/foundationdb/ReadTransaction.java

```diff
@@ -455,6 +455,26 @@
 	 */
 	CompletableFuture<Long> getEstimatedRangeSizeBytes(Range range);
 
+	/**
+	 * Gets a list of keys that can split the given range into similar sized chunks based on chunkSize
```

Comment:
We should document that this includes the begin and end key.

Reply:
Now that I think about the decision to include begin and end keys in native api when returning the result, it might be better not to do so? Cause if I were the user and I call this API, I'd assume it will return the split points only. So an empty array would suggest that it's not splittable. What do you think?

Reply:
I could see an argument either way, do we have a candidate user of this feature that we could ask their opinion?

Reply:
@alecgrieser Would this feature be useful to you?